build_debian.sh

```diff
@@ -208,6 +208,23 @@ sudo LANG=C chroot $FILESYSTEM_ROOT apt-get update
 sudo LANG=C chroot $FILESYSTEM_ROOT apt-get -y install docker-ce=${DOCKER_VERSION}
 sudo LANG=C chroot $FILESYSTEM_ROOT apt-get -y remove software-properties-common gnupg2
 
+if [ "$INSTALL_KUBERNETES" == "y" ]
+then
+    KUBERNETES_VERSION="${KUBERNETES_VERSION:-1.18.0-00}"
+    ## Install Kubernetes
+    echo '[INFO] Install kubernetes'
+    sudo https_proxy=$https_proxy LANG=C chroot $FILESYSTEM_ROOT curl -o /tmp/k8s.gpg -fsSL https://packages.cloud.google.com/apt/doc/apt-key.gpg
+    sudo LANG=C chroot $FILESYSTEM_ROOT apt-key add /tmp/k8s.gpg
+    sudo LANG=C chroot $FILESYSTEM_ROOT rm /tmp/k8s.gpg
```

Comment:
>rm /tmp/k8s.gpg [](start = 40, length = 15)

No need to create tmp file. Use `curl -fsSL [https://packages.cloud.google.com/apt/doc/apt-key.gpg](https://packages.cloud.google.com/apt/doc/apt-key.gpg) | sudo apt-key add -`

Reply:
Sure

Reply:
Did you try? What is the error?

---
In reply to: [404418653](https://github.com/Azure/sonic-buildimage/pull/4374#discussion_r404418653) [](ancestors = 404418653)

Reply:
Let me take back my comments. It worked. The failure was in k8s packages download, but for different reason. I will fix.